esp-hal-common/src/spi/master.rs

```diff
@@ -2708,23 +2708,25 @@ pub trait Instance {
     // FIXME: Using something like `core::slice::from_raw_parts` and
     // `copy_from_slice` on the receive registers works only for the esp32 and
     // esp32c3 varaints. The reason for this is unknown.
+    #[inline]
```

Comment:
So I guess the comment above this attribute is no longer right?

Reply:
I think so, since the S3 and ESP32 should have the same processor?